tool/autoqa/sgd/normalize-data.js

```diff
@@ -136,13 +89,13 @@ class Normalizer {
             } else if (expectedType.type === 'tt:Number') {
                 return parseFloat(value);
             } else if (expectedType.type === 'tt:Duration') {
-                return parseDuration(value);
+                return ThingTalk.Units.transformToBaseUnit(parseFloat(value), 'min');
             } else if (expectedType.type === 'tt:Measure') {
-                return parseMeasure(value);
+                throw new Error(`No measurement type in sgd dataset`);
             } else if (expectedType.type.startsWith('tt:Enum(')) {
                 const enumerands = expectedType.type.substring('tt:Enum('.length, expectedType.type.length - 1).split(/,/g);
                 value = cleanEnumValue(value);
-                if (value === undefined)
+                if (value === undefined || value === 'Dontcare')
```

Comment:
Do we need to handle Dontcare values correctly (map to a BooleanExpression.DontCare)?

Reply:
Yes, but in step we are generating normalized data, so this should not matter? But yeah, we need to take care of this when transforming their dataset to our thingtalk annotation